packages/@tinacms/schema-tools/src/schema/resolveField.ts

```diff
@@ -77,11 +77,26 @@ export const resolveField = (
             options: field.options,
           }
         }
+        if (
+          field.options[0] &&
+          typeof field.options[0] === 'object' &&
+          field.options[0].icon
+        ) {
+          return {
+            component: 'button-toggle',
+            ...field,
+            ...extraFields,
+            options: field.options,
+          }
+        }
         return {
           component: 'select',
           ...field,
           ...extraFields,
-          options: [{ label: `Choose an option`, value: '' }, ...field.options],
+          options:
+            field.ui && field.ui.component !== 'select'
```

Comment:
@spbyrne What is the motivation for adding this check for component !== 'select' ?

Reply:
Things like radio buttons & checkbox groups shouldn't have an option labelled 'Choose an option', but it makes sense in the case of select otherwise the first option would be displayed even though it's not technically selected.

Reply:
Makes sense!